src/frontend/components/Dashboard/index.tsx

```diff
@@ -31,7 +35,7 @@ const DashboardContainer = styled.div`
 `;
 
 /** Props shape for the Dashboard component. */
-export interface DashboardProps {
+interface DashboardProps {
```

Comment:
`BaseDashboardProps` ?

Reply:
The interface are the same between the connection and the component. This is why I kept only one. But maybe I'm wrong.

Reply:
Oh that's okay then, my bad